src/tree/values/nodes/StructuredText.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Represents a piece of structure that is present in the original text, but is not relevant for the further analysis
+ * of the text.
+ *
+ * Talking about HTML, this would encompass thing like <div>, <section>, <aside>, <fieldset> and other HTML block elements.
+ */
+class StructuredText {
+	/**
+	 * Represents a piece of structure that is present in the original text, but is not relevant for the further
+	 * analysis of the text.
+	 *
+	 * Talking about HTML, this would encompass thing like <div>, <section>, <aside>, <fieldset> and other HTML block elements.
+	 *
+	 * @param {Paragraph|List|Heading[]} children	The sub-elements of the structured text.
```

Comment:
Maybe I am incorrect, but I think this should be 
```suggestion
	 * @param {Paragraph[]|List[]|Heading[]} children	The sub-elements of the structured text.
```